package.json

```diff
@@ -7,6 +7,7 @@
     "test": "jest",
     "docs-dev": "webpack-dev-server --mode=development --port 9398 --config ./src/docs/webpack.config.js",
     "build-docs": "webpack --mode=production --config ./src/docs/webpack.config.js && node ./scripts/buildDocs.js",
+    "build-new-docs": "docz dev",
```

Comment:
We should normalize all of the scripts and clean up the old scripts.

- `build-new-docs` -> `docs:start`
- add `docs:build` (producing static html)
- add `docs:publish`: `npm run docs:build && npm run gh-pages`
- update gh-pages to use the directory `docs:build` produces, but don't run it yet